contracts/StablecoinConverter.sol

```diff
@@ -293,10 +294,12 @@ contract StablecoinConverter is EpochTokenLocker {
         uint256 burntFees = uint256(tokenConservation[0]) / 2;
         require(utility + burntFees > disregardedUtility, "Solution must be better than trivial");
         // burntFees ensures direct trades (when available) yield better solutions than longer rings
-        checkAndOverrideObjectiveValue(utility - disregardedUtility + burntFees);
+        uint256 objectiveValue = utility - disregardedUtility + burntFees;
```

Comment:
Would this be a good place to introduce the use of `SafeMath`?

Reply:
Addressed, note that we don't need to use safe match on the objective value calculation line because of the require statement above. We just need to make sure that `totalUtility + burntFees` can't overflow

Reply:
I feel like it wouldn't hurt to add the safemath here though. In a future change the require statement might go away (e.g. we might allow solutions with slight negative combined utility) and might not update the objectiveValue calculation.

Reply:
This is done in a recent PR. Let me find it.

https://github.com/gnosis/dex-contracts/pull/301

Happy to make it a separate change if you'd prefer.